tests/fpsemi-examples.cpp

```diff
@@ -1052,17 +1052,17 @@ namespace libsemigroups {
               {{3, 1, 1, 2, 1}, {3, 1, 0, 2}},
               {{1, 2, 1, 2, 0, 2}, {2, 1, 2, 0, 2}}};
     } else if (n >= 4 && val == author::Aizenstat) {
-      LIBSEMIGROUPS_EXCEPTION("not yet implemented");
-      // FIXME this is missing the relations from T_n
```

Comment:
Looking at this quickly just now, have you maybe made the same mistake as me, forgotten to add the relations from the full transformation monoid mentioned in this FIXME?

Reply:
Ah I have misread this, in a way I can't even see how I managed to -- I will fix now, thanks!